tracer/build/PackageVersionsLatestMajors.g.props

```diff
@@ -738,13 +738,13 @@ NOTE:  This code was generated by the GeneratePackageVersions tool. To safely
       <RequiresDockerDependency>True</RequiresDockerDependency>
     </PackageVersionSample>
     <PackageVersionSample Include="test\test-applications\integrations\Samples.MySqlConnector\Samples.MySqlConnector.csproj">
-      <Properties>ApiVersion=2.1.9;RestoreRecursive=false;BuildProjectReferences=false</Properties>
+      <Properties>ApiVersion=2.1.10;RestoreRecursive=false;BuildProjectReferences=false</Properties>
       <TargetFramework>net461</TargetFramework>
       <RequiresDockerDependency>True</RequiresDockerDependency>
     </PackageVersionSample>
     <PackageVersionSample Include="test\test-applications\integrations\Samples.MySqlConnector\Samples.MySqlConnector.csproj">
       <Properties>ApiVersion=0.69.10;RestoreRecursive=false;BuildProjectReferences=false</Properties>
-      <TargetFramework>netcoreapp3.0</TargetFramework>
+      <TargetFramework>netcoreapp2.1</TargetFramework>
       <RequiresDockerDependency>True</RequiresDockerDependency>
     </PackageVersionSample>
```

Comment:
um? @pierotibou? 🤔

Reply:
wow, 🤔 , how did I do that? 
I used the UpdateVersion target and I don't recall doing manual modifs. I must have obviously, but I don't see when/how yet.